backends/platform/sdl/sdl.cpp

```diff
@@ -313,11 +319,15 @@ void OSystem_SDL::initBackend() {
 }
 
 #if defined(USE_OPENGL_GAME) || defined(USE_OPENGL_SHADERS)
-void OSystem_SDL::detectFramebufferSupport() {
+void OSystem_SDL::detectOpenGLFeaturesSupport() {
+	_oglType = OpenGL::kOGLContextNone;
 	_supportsFrameBuffer = false;
+	_supportsShaders = false;
 #if USE_FORCED_GLES2
-	// Framebuffers are always available with GLES2
+	// Framebuffers and shaders are always available with GLES2
+	_oglType = kOGLContextGLES2;
```

Comment:
This change causes my build to fail with
`backends/platform/sdl/sdl.cpp:328:13: error: use of undeclared identifier 'kOGLContextGLES2'; did you mean 'OpenGL::kOGLContextGLES2'?`

Not sure this is an issue with my headers in the emscripten backend. But seeing that it's defined with OpenGL:: 5 lines above, is it possible this was an error?

Reply:
It was. Tanks to @criezy for the fix.
Sorry!